docs/articles_en/about_openvino/system_requirements.rst

```diff
@@ -32,13 +32,13 @@ CPU
 
       * Ubuntu 22.04 long-term support (LTS), 64-bit (Kernel 5.15+)
       * Ubuntu 20.04 long-term support (LTS), 64-bit (Kernel 5.15+)
-      * Ubuntu 18.04 long-term support (LTS) with limitations, 64-bit (Kernel 5.4+)
+      * Ubuntu 18.04 long-term support (LTS) with limitations, 64-bit (Kernel 5.4+), ARM64
```

Comment:
since we have specified architecture here, maybe we need to specify it in other places as well to denote explicit `x86_64` ?